R/summary_rows.R

```diff
@@ -393,28 +393,19 @@ summary_rows <- function(
   # Pull a character vector of available groups from `groups_rows_tbl`
   available_groups <- dplyr::pull(groups_rows_tbl, group_id)
 
-  # Return data unchanged if there are no groups and this summary is not
-  # a grand summary
-  if (length(available_groups) < 1) {
-    if (!(
-      is.character(groups) &&
-      length(groups) == 1 &&
-      groups == ":GRAND_SUMMARY:"
-    )) {
-      return(data)
-    }
-  }
-
   # Resolve the group names
   groups <-
     resolve_groups(
       expr = {{ groups }},
       vector = available_groups
     )
```

Comment:
Moving the resolver before has forced me to add the `isFALSE()` condition in the resolver. The `groups = FALSE` option is undocumented in `summary_rows()` groups argument, but I wonder if I missed some other edge cases?

Reply:
I think you might be fine here.